exporter/prometheusexporter/README.md

```diff
@@ -17,6 +17,8 @@ The following settings can be optionally configured:
 - `send_timestamps` (default = `false`): if true, sends the timestamp of the underlying
   metric sample in the response.
 - `metric_expiration` (default = `5m`): defines how long metrics are exposed without updates
+- `resource_attributes_as_tag` (default = `false`): if set to true will transform all resource
```

Comment:
I was looking into this exact same feature with the `prometheusremotewriteexporter`. Was just wondering - strictly speaking it is "resource attributes as labels"? I guess Prometheus uses the term  "label" instead of "tag"?

Reply:
As @bogdandrutu I'll switch it to a convention thats used over in the contrib repo so things are kept consistent, so this will change too 

``` yaml
        resource_to_telemetry_conversion:
            enabled: true
```